core/src/test/java/com/linecorp/armeria/server/ServerTest.java

```diff
@@ -331,6 +331,26 @@ public void duplicatedPort() {
                 .hasCauseInstanceOf(IOException.class);
     }
 
+    @Test
+    public void testActiveLocalPort() throws Exception {
+        final Server server = ServerTest.server.server();
+        int expectedPort = server.activePort().get().localAddress().getPort();
+        assertThat(server.activeLocalPort()).isEqualTo(expectedPort);
+
+        server.stop().get();
+        assertThatThrownBy(server::activeLocalPort)
+                .isInstanceOf(IllegalStateException.class)
+                .hasMessageContaining("no active local ports");
+
+        final Server noPortsServer = new ServerBuilder()
+                .service("/", (ctx, res) -> HttpResponse.of(""))
+                .build();
+
+        assertThatThrownBy(noPortsServer::activeLocalPort)
+                .isInstanceOf(IllegalStateException.class)
+                .hasMessageContaining("no active local ports");
```

Comment:
Last comment - Could you also test `activeLocalPort(SessionProtocol)`? This test method tests `activeLocalPort()` only.

Reply:
@trustin i changed some tests! :)